pygtfs/gtfs_entities.py

```diff
@@ -211,7 +216,31 @@ class Route(Base):
     trips = relationship("Trip", backref="route")
     fare_rules = relationship("FareRule", backref="route")
 
-    _validate_route_type = _validate_int_choice(range(8), 'route_type')
+    valid_route_types = range(8)
+    # https://developers.google.com/transit/gtfs/reference/extended-route-types
+    valid_extended_route_types = [
+        range(8),
+        range(100, 118),
+        range(200, 210),
+        [300],
+        range(400, 406),
+        [500],
+        [600],
+        range(700, 717),
+        [800],
+        range(900, 907),
+        range(1000, 1022),
+        range(1100, 1115),
+        [1200],
+        range(1300, 1308),
+        range(1400, 1403),
+        range(1500, 1508),
+        range(1600, 1605),
+        range(1700, 1703)
+    ]
+    # flatten the list of lists to a list
+    valid_extended_route_types = [item for sublist in valid_extended_route_types for item in sublist]
```

Comment:
in python3 this will also evaluate all the range objects. Since there are just 127 integers here, that's totally fine with me.  The error message of the validation will also be pretty long, but again, not too bad. The alternative, a validation function checking `in` (`__contains__`) for each object is overly complicated and uncalled for. So yeah this solutions looks ok to me.